src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ItemTests.cs

```diff
@@ -148,5 +148,14 @@ public void TestAzureRSVaultMSI()
             TestController.NewInstance.RunPsTest(
                 _logger, PsBackupProviderTypes.IaasVm, "Test-AzureRSVaultMSI");
         }
+
+        [Fact(Skip = "to be fixed in upcoming release")]
```

Comment:
Test case should not be skipped. Is there any way you can unblock it?

Reply:
Currently we're facing a service side bug. Once fixed I'll add this test. Currently I see no way to unblock this test.